core/src/main/java/org/stellar/anchor/util/TestUtils.java

```diff
@@ -0,0 +1,38 @@
+package org.stellar.anchor.util;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class TestUtils {
```

Comment:
💡 Although this is cool, unit-testing private methods. ([some reference](https://stackoverflow.com/questions/34571/how-do-i-test-a-class-that-has-private-methods-fields-or-inner-classes) which I also agree.)

If we wish to unit-test a method, we can make the method package private and have the unit test reside in the same package as the target class.

Reply:
Fixed in https://github.com/stellar/java-stellar-anchor-sdk/pull/448/commits/85c61df17d3b6a4e3edd3c1bb9994b370174a982.